.generated_docs

```diff
@@ -23,12 +23,14 @@ docs/man/man1/kubectl-convert.1
 docs/man/man1/kubectl-create.1
 docs/man/man1/kubectl-delete.1
 docs/man/man1/kubectl-describe.1
+docs/man/man1/kubectl-drain.1
 docs/man/man1/kubectl-edit.1
 docs/man/man1/kubectl-exec.1
 docs/man/man1/kubectl-explain.1
 docs/man/man1/kubectl-expose.1
 docs/man/man1/kubectl-get.1
 docs/man/man1/kubectl-label.1
+docs/man/man1/kubectl-lame.1
```

Comment:
disable-node and enable-node are also clearer ways of indicating the
purpose of the commands.  Agree that lame is probably not a good verb to
use in common actions like this.

On Nov 23, 2015, at 5:33 PM, Derek Carr <notifications@github.com> wrote:

In .generated_docs
<https://github.com/kubernetes/kubernetes/pull/16698#discussion_r45671140>:

>  docs/man/man1/kubectl-edit.1
>  docs/man/man1/kubectl-exec.1
>  docs/man/man1/kubectl-explain.1
>  docs/man/man1/kubectl-expose.1
>  docs/man/man1/kubectl-get.1
>  docs/man/man1/kubectl-label.1
> +docs/man/man1/kubectl-lame.1

+1 @pmorie <https://github.com/pmorie> feedback.

I prefer rotate-node-out and rotate-node-in since I don't see this command
taking things that are not nodes in the future.

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/16698/files#r45671140>.